app/views/errors/unauthorized.html.haml

```diff
@@ -0,0 +1,7 @@
+= content_for :title do
+  You are not authorised to access this page (401)
```

Comment:
authori**z**ed or authori**s**ed? With this change we now have both. Since we're .au we should consistently use **s**, so I suggest we change to that. (in a new card if you dont want to do it here)